python/paddle/v2/framework/tests/test_linear_chain_crf_op.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+import random
+import numpy as np
+
+from op_test import OpTest
+
+
+class LinearChainCrfForward(object):
```

Comment:
A test case of LinearChainCrfForward() should also be included to make sure the implementation is correct. Furthermore, it is also useful for detecting unexpected errors introduced by future modifications. Reference: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/gserver/tests/test_CRFLayerGrad.cpp#L56